resources/views/applicant/application_index/application.html.twig

```diff
@@ -57,7 +57,16 @@
             <div data-c-grid-item="base(1of2) tl(1of6)" data-c-align="base(center)">
                 {% if item.application_status.name == "draft" %}
                     {% set expired = item.job_poster.close_date_time.isPast() %}
-                    <a data-c-button="solid(c1)" data-c-radius="rounded" href={{ expired ? route('applications.show', item) : route('job.application.edit.1', item.job_poster) }} title="{{ expired ? application_index.view_link_title : application_index.draft_link_title }}">{{ expired ? application_index.view_link_label : application_index.draft_link_label }}</a>
+                    <a
+                        data-c-button="solid(c1)"
+                        data-c-radius="rounded"
+                        href={{ expired
+                            ? route('applications.show', item)
+                            : item.version_id is same as(2)
```

Comment:
Probably fine as-is, but might be worth passing the `$version2id` through to the view for comparison here